src/pixi/core/math/ObservablePoint.hx

```diff
@@ -0,0 +1,35 @@
+package pixi.core.math;
+
+@:native("PIXI.ObservablePoint")
+extern class ObservablePoint extends Point {
+
+	/**
+	 * The Point object represents a location in a two-dimensional coordinate system, where x represents
+	 * the horizontal axis and y represents the vertical axis.
+	 * An observable point is a point that triggers a callback when the point's position is changed.
+	 *
+	 * @class
+	 * @memberof PIXI
+	 * @param cb {Function} callback when changed
+	 * @param scope {Object} owner of callback
+	 * @param [x=0] {number} position of the point on the x axis
+	 * @param [y=0] {number} position of the point on the y axis
+	 */
+	function new(cb:Dynamic, scope:Dynamic, x:Float, y:Float);
```

Comment:
Ok, just noticed the optionals now! :)

About the Dynamic, i thought about putting Void->Void but wouldn't disallowing parameters reduce the options for the callback too much?

I'm gonna commit the fixes, thanks!